README.md

```diff
@@ -33,11 +33,12 @@ As an example, to run the latest version of the watch-tower via `docker`:
 docker run --rm -it \
   ghcr.io/cowprotocol/watch-tower:latest \
   run \
-  --rpc <rpc-url> \
-  --deployment-block <deployment-block> \
+  --chain-config <rpc>,<deployment-block>,<watchdog-timeout> \
   --page-size 5000
 ```
 
+**NOTE**: `watchdog-timeout` above is _optional_. If not specified, the default is 30 seconds. This is the timeout from which if the watch-tower does not receive a new block, it will exit, or signal non-live to Kubernetes. This is used to restart the watch-tower in the event of a RPC / network failure.
+
```

Comment:
There's one more optional chain-config parameter not mentioned here in the read-me: orderbook api url

Reply:
In order to avoid just repeating the help written for commander, notes were added directing the user to `--help`.